Source/Forms/Gen4/Wild4.cpp

```diff
@@ -465,6 +466,16 @@ void Wild4::searcherEncounterIndexChanged(int index)
             break;
         }
 
+        if ((currentProfile->getVersion() & Game::HGSS) != Game::None && index > 2)
+        {
+            ui->comboBoxSearcherLead->removeItem(ui->comboBoxSearcherLead->findData(toInt(Lead::SuctionCups)));
+            ui->comboBoxSearcherLead->addItem(tr("Suction Cups"), toInt(Lead::SuctionCups));
```

Comment:
Same deal as above. Only add if it doesn't exist

Reply:
done